INSTALL

```diff
@@ -45,11 +45,11 @@ pip install . --constraint requirements/requirements-python3.7.txt
 # START EXTRAS HERE
 
 all, all_dbs, async, atlas, aws, azure, cassandra, celery, cgroups, cloudant, dask, databricks,
-datadog, devel, devel_ci, devel_hadoop, doc, docker, druid, elasticsearch, exasol, facebook, gcp,
```

Comment:
This is a breaking change. Can you keep old for backward compatibility? Similar to `gcp_api`

Reply:
Right. I thought it was not realeased in 1.10 but it was. Fixing.